src/handlers/console.ts

```diff
@@ -32,6 +33,24 @@ export class ConsoleHandler implements IDisposable {
       this.promptCellCreated,
       this
     );
+
+    const toolbar = new Toolbar();
+    toolbar.addItem(
+      'debugger-lifeCycle-button',
+      new ToolbarButton({
+        className: 'jp-debugger-switch-button',
+        iconClassName: 'jp-toggle-switch',
+        onClick: () => {
+          this.toogleState = !this.toogleState;
+          this.consolePanel.node.setAttribute(
+            'data-debugger-on',
+            this.toogleState.toString()
+          );
+        },
+        tooltip: 'ON/OFF Debugger state'
```

Comment:
Maybe something like "Enable / Disable Debugger" would be more user-friendly?

Reply:
You definitely, absolutely have right.